xbmc/Application.cpp

```diff
@@ -3628,6 +3628,7 @@ void CApplication::OnPlayBackStarted()
 
   CGUIMessage msg(GUI_MSG_PLAYBACK_STARTED, 0, 0);
   g_windowManager.SendThreadMessage(msg);
+  CStereoscopicsManager::GetInstance().PlaybackStarted();
```

Comment:
should that one be called before the message is propagated to the windowManager in order to have as less delay as possible? Or are you going to remove this call here and instead call the manager from the player at an earlier stage?